modules/virt-querying-metrics.adoc

```diff
@@ -12,14 +12,17 @@ The following metric descriptions include example Prometheus Query Language (Pro
 These metrics are not an API and might change between versions.
 ====
 
-
 [id="virt-promql-vcpu-metrics_{context}"]
 == vCPU metrics
 
+.Prerequisite
+
+* The vCPU metric requires the `schedstats=enable` kernel argument applied to the `MachineConfig` object before it can be used. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. See the xref:../../post_installation_configuration/machine-configuration-tasks.adoc#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[{product-title} machine configuration tasks] documentation for more information on applying a kernel argument.
```

Comment:
Because this is not a procedure module, rather than using the term 'Prerequisite', maybe format it as a 'Note'. You can place this note after the vCPU query description and before the Example query.